### ISSUE TITLE
facet plots: strip.text.y & strip.position = 'left' not accepting angle argument

### ISSUE THREAD

**Issue by sebdunnett** [tidyverse/ggplot2#3888]
Hi, thanks for all your work on ggplot2; first issue report for me. I recently updated to ggplot2 3.3.0 and my code to produce facet maps with facet strips on the left no longer seems able to produce text the right way up. Reprex below

```r
ggplot(mpg, aes(displ, hwy)) +
  geom_point() +
  facet_wrap(~cyl, strip.position = 'right') +
  theme(strip.text.y = element_text(angle=45))

ggplot(mpg, aes(displ, hwy)) +
  geom_point() +
  facet_wrap(~cyl, strip.position = 'left') +
  theme(strip.text.y = element_text(angle=45))
```

**Comment by batpigandme** [comment]
Just adding a reprex with the rendered plots
``` r
library(tidyverse)
ggplot(mpg, aes(displ, hwy)) +
  geom_point() +
  facet_wrap(~cyl, strip.position = 'right') +
  theme(strip.text.y = element_text(angle=45))
```

![](https://i.imgur.com/4kqYVgQ.png)

``` r

ggplot(mpg, aes(displ, hwy)) +
  geom_point() +
  facet_wrap(~cyl, strip.position = 'left') +
  theme(strip.text.y = element_text(angle=45))
```

![](https://i.imgur.com/6JQYie7.png)

<sup>Created on 2020-03-11 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0.9001)</sup>

**Comment by clauswilke** [comment]
This works.
``` r
library(ggplot2)

ggplot(mpg, aes(displ, hwy)) +
  geom_point() +
  facet_wrap(~cyl, strip.position = 'left') +
  theme(strip.text.y.left = element_text(angle=45))
```

![](https://i.imgur.com/doUd71E.png)

<sup>Created on 2020-03-11 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by batpigandme** [comment]
I suspected as much, but it wasn't in autosuggest, so I thought it didn't exist 🙃

**Comment by clauswilke** [comment]
Not sure whether the other code should work also. I'll leave that for somebody else to decide.

**Comment by thomasp85** [comment]
I think the main “issue” is that the vertical strips have different rotation for left and right by default and as such you have to overwrite it at that level for it to go into effect.

**Comment by sebdunnett** [comment]
Thank you @clauswilke , I didn't know about strip.text.y.left.

**Comment by rbreejen** [comment]
I am seeing the same issue with facet_grid. The accepted answer provided in [this](https://stackoverflow.com/questions/40484090/rotate-switched-facet-labels-in-ggplot2-facet-grid) Stackoverflow question does not work anymore.

Code: 
`library(ggplot2)
df <- group_by(mpg, manufacturer) %>%
  summarise(cty = mean(cty), hwy = mean(hwy)) %>%
  ungroup()

df <- melt(df, id.vars = "manufacturer")

ggplot() +
  geom_bar(data =df, aes(x = variable, y = value), stat = "identity") +
  facet_grid(manufacturer ~ ., switch = "y")+
 theme(strip.text.y = element_text(angle = 180))`

**Comment by clauswilke** [comment]
Yes, this change was introduced deliberately in #3683. I think we didn't mention this in the changelog and we should have. I have edited the SO answer so it's correct now.

@rbreejen Note that the previous working code had an obvious flaw: You had to specify `angle = 180` to get an angle of 0, because the code internally was manipulating the rotation angle when switching the location of the strips. That was confusing, and also inconsistent with how axes work.

@thomasp85 Can we retroactively add a news item to the 3.3.0 release? Or should we add it to 3.3.1 but point out the change happened already in 3.3.0? The second seems cleaner to me.

**Comment by thomasp85** [comment]
Let’s add it to 3.3.1

**Comment by pabgamu** [comment]
Thanks for this fix, I have been reviewing both this and #3896 tickets and I think it should make sense to add `strip.text.y.left/right` and `strip.text.x.top/bottom` to theme inputs for consistency sake, similar to `axis.title.x.top`, etc., I think this will be very helpful for any user, specially new ones,  as I think these inputs are only mentioned in news file, please let me know if I am wrong about this. 
I can open new ticket or submit pull request if more convenient, just let me know, thanks.
